### PR TITLE
[WIP] Fix ui test

### PR DESCRIPTION
**Pull request by jameswnl**
Thanks to @psav , got some progress in getting the selenium-based UI tests to move forward with this PR.

It us still failing ultimately with
```
2021-05-06 13:58:51 ERROR [UI-NAV/ApplicationPlatformUI/LoggedIn]: pre_nav_prep hit Exception: Message: Could not find an element Locator(by='xpath', locator=".//button[@id='login-show-step2']")
Traceback (most recent call last):
  File "/iqe_venv/lib64/python3.6/site-packages/iqe/base/application/implementations/web_ui.py", line 365, in pre_nav_prep
    return fn(*args, **kwargs)
  File "/iqe_venv/lib64/python3.6/site-packages/iqe_platform_ui/__init__.py", line 177, in step
    self.obj.login()
  File "/iqe_venv/lib64/python3.6/site-packages/iqe_platform_ui/__init__.py", line 115, in login
    view.do_login(self.application.user.username, self.application.user.password)
  File "/iqe_venv/lib64/python3.6/site-packages/widgetastic/widget/base.py", line 81, in wrapped
    return method(self, *new_args, **new_kwargs)
  File "/iqe_venv/lib64/python3.6/site-packages/iqe_platform_ui/views/__init__.py", line 289, in do_login
    self.next_button.click()
  File "/iqe_venv/lib64/python3.6/site-packages/widgetastic/log.py", line 115, in wrapped
    result = f(self, *args, **kwargs)
  File "/iqe_venv/lib64/python3.6/site-packages/widgetastic/widget/base.py", line 695, in click
    self.browser.click(self, ignore_ajax=(handle_alert is not None))
  File "/iqe_venv/lib64/python3.6/site-packages/widgetastic/utils.py", line 695, in wrap
    return method(*args, **kwargs)
  File "/iqe_venv/lib64/python3.6/site-packages/widgetastic/browser.py", line 400, in click
    el = self.move_to_element(locator, *args, **kwargs)
  File "/iqe_venv/lib64/python3.6/site-packages/iqe/base/application/implementations/web_ui.py", line 198, in move_to_element
    el = self._browser.element(locator, parent=parent)
  File "/iqe_venv/lib64/python3.6/site-packages/widgetastic/browser.py", line 377, in element
    elements = self.elements(locator, *args, **kwargs)
  File "/iqe_venv/lib64/python3.6/site-packages/widgetastic/utils.py", line 695, in wrap
    return method(*args, **kwargs)
  File "/iqe_venv/lib64/python3.6/site-packages/widgetastic/browser.py", line 278, in elements
    locator = self._process_locator(locator)
  File "/iqe_venv/lib64/python3.6/site-packages/widgetastic/browser.py", line 215, in _process_locator
    return locator.__element__()
  File "/iqe_venv/lib64/python3.6/site-packages/widgetastic/widget/base.py", line 81, in wrapped
    return method(self, *new_args, **new_kwargs)
  File "/iqe_venv/lib64/python3.6/site-packages/widgetastic/widget/base.py", line 371, in __element__
    return self.parent_browser.element(locator)
  File "/iqe_venv/lib64/python3.6/site-packages/widgetastic/browser.py", line 382, in element
    ) from None
selenium.common.exceptions.NoSuchElementException: Message: Could not find an element Locator(by='xpath', locator=".//button[@id='login-show-step2']")
```

### CODE REVIEW

File: cicd/iqe_pod/create_iqe_pod.py [hunk 1]
```diff
@@ -187,7 +218,7 @@ def _create_conf_secret(namespace):
         "metadata": {"name": SECRET_NAME},
         "data": {"settings.local.yaml": encoded_conf},
     }
-    oc("create", f="-", n=namespace, _in=json.dumps(secret))
+    oc("apply", f="-", n=namespace, _in=json.dumps(secret))
```
Comment (jameswnl) [inline cicd/iqe_pod/create_iqe_pod.py:221]: This change is not necessary, just that using `apply` is more useful when doing a repeated deploy from command line. 

Same with L204 `_create_pod`

File: cicd/iqe_pod/create_iqe_pod.py [hunk 2]
```diff
@@ -53,6 +71,19 @@ def _build_test_conf(env_parser):
     env_name = "clowder_smoke"
     env_conf = conf[env_name] = {}
 
+    env_conf["main"] = {
+        "hostname": "front-end-aggregator-ephemeral-20.apps.c-rh-c-eph.8p0c.p1.openshiftapps.com",
```
Comment (jameswnl) [inline cicd/iqe_pod/create_iqe_pod.py:75]: This `hostname` would need a `oc get route front-end-aggregator -o jsonpath='https://{.spec.host}{"\n"}'` equivalent to update it at runtime.

**Comment by jameswnl** [comment]
@bsquizz fix to that failure might be easy for you :)

**Comment by bsquizz** [comment]
I think at this point all of this is pretty deprecated w/ the move to the CJI

**Pull request closed without merging**